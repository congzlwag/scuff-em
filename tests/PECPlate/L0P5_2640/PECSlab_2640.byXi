# scuff-caspol running on (null) (05/04/20::16:42:13)
#
# command line:
#
# scuff-caspol  --geometry  PECSlab_2640.scuffgeo  --EPFile  EPFile4
#  --XiFile  ../XiFile2  --Atom  Rb
# 
#
# data columns:
#1: X (um) 
#2: Y (um) 
#3: Z (um) 
#4: Xi (imaginary frequency) (3e14 rad/sec)
#5: polarizability of Rb (a_0^3)
#6: CP potential per unit frequency for Rb (neV / (3e14 rad/sec))
# scuff-caspol running on (null) (05/04/20::17:15:29)
#
# command line:
#
# scuff-caspol  --geometry  PECSlab_2640.scuffgeo  --EPFile  EPFile4
#  --XiFile  ../XiFile2  --Atom  Rb
# 
#
# data columns:
#1: X (um) 
#2: Y (um) 
#3: Z (um) 
#4: Xi (imaginary frequency) (3e14 rad/sec)
#5: polarizability of Rb (a_0^3)
#6: CP potential per unit frequency for Rb (neV / (3e14 rad/sec))
# scuff-caspol running on (null) (05/04/20::20:54:03)
#
# command line:
#
# scuff-caspol  --geometry  PECSlab_2640.scuffgeo  --EPFile  EPFile4
#  --Atom  Rb  --XiFile  ../XiFile2
# 
#
# data columns:
#1: X (um) 
#2: Y (um) 
#3: Z (um) 
#4: Xi (imaginary frequency) (3e14 rad/sec)
#5: polarizability of Rb (a_0^3)
#6: CP potential per unit frequency for Rb (neV / (3e14 rad/sec))
# scuff-caspol running on (null) (05/04/20::21:52:51)
#
# command line:
#
# scuff-caspol  --geometry  PECSlab_2640.scuffgeo  --EPFile  EPFile4
#  --Atom  Rb  --XiFile  ../XiFile2
# 
#
# data columns:
#1: X (um) 
#2: Y (um) 
#3: Z (um) 
#4: Xi (imaginary frequency) (3e14 rad/sec)
#5: polarizability of Rb (a_0^3)
#6: CP potential per unit frequency for Rb (neV / (3e14 rad/sec))
# scuff-caspol running on (null) (05/04/20::22:15:25)
#
# command line:
#
# scuff-caspol  --geometry  PECSlab_2640.scuffgeo  --EPFile  EPFile4
#  --Atom  Rb  --XiFile  ../XiFile2
# 
#
# data columns:
#1: X (um) 
#2: Y (um) 
#3: Z (um) 
#4: Xi (imaginary frequency) (3e14 rad/sec)
#5: polarizability of Rb (a_0^3)
#6: CP potential per unit frequency for Rb (neV / (3e14 rad/sec))
# scuff-caspol running on (null) (05/04/20::22:18:47)
#
# command line:
#
# scuff-caspol  --geometry  PECSlab_2640.scuffgeo  --EPFile  EPFile4
#  --Atom  Rb  --XiFile  ../XiFile2
# 
#
# data columns:
#1: X (um) 
#2: Y (um) 
#3: Z (um) 
#4: Xi (imaginary frequency) (3e14 rad/sec)
#5: polarizability of Rb (a_0^3)
#6: CP potential per unit frequency for Rb (neV / (3e14 rad/sec))
# scuff-caspol running on (null) (05/05/20::00:22:36)
#
# command line:
#
# scuff-caspol  --geometry  PECSlab_2640.scuffgeo  --EPFile  EPFile4
#  --Atom  Rb  --XiFile  ../XiFile2
# 
#
# data columns:
#1: X (um) 
#2: Y (um) 
#3: Z (um) 
#4: Xi (imaginary frequency) (3e14 rad/sec)
#5: polarizability of Rb (a_0^3)
#6: CP potential per unit frequency for Rb (neV / (3e14 rad/sec))
# scuff-caspol running on (null) (05/05/20::00:27:18)
#
# command line:
#
# scuff-caspol  --geometry  PECSlab_2640.scuffgeo  --EPFile  EPFile4
#  --Atom  Rb  --XiFile  ../XiFile2
# 
#
# data columns:
#1: X (um) 
#2: Y (um) 
#3: Z (um) 
#4: Xi (imaginary frequency) (3e14 rad/sec)
#5: polarizability of Rb (a_0^3)
#6: CP potential per unit frequency for Rb (neV / (3e14 rad/sec))
# scuff-caspol running on (null) (05/05/20::09:44:23)
#
# command line:
#
# scuff-caspol  --geometry  PECSlab_2640.scuffgeo  --EPFile  EPFile4
#  --Atom  Rb  --XiFile  ../XiFile2
# 
#
# data columns:
#1: X (um) 
#2: Y (um) 
#3: Z (um) 
#4: Xi (imaginary frequency) (3e14 rad/sec)
#5: polarizability of Rb (a_0^3)
#6: CP potential per unit frequency for Rb (neV / (3e14 rad/sec))
# scuff-caspol running on (null) (05/05/20::10:19:36)
#
# command line:
#
# scuff-caspol  --geometry  PECSlab_2640.scuffgeo  --EPFile  EPFile4
#  --Atom  Rb  --XiFile  ../XiFile2
# 
#
# data columns:
#1: X (um) 
#2: Y (um) 
#3: Z (um) 
#4: Xi (imaginary frequency) (3e14 rad/sec)
#5: polarizability of Rb (a_0^3)
#6: CP potential per unit frequency for Rb (neV / (3e14 rad/sec))
# scuff-caspol running on (null) (05/05/20::10:30:19)
#
# command line:
#
# scuff-caspol  --geometry  PECSlab_2640.scuffgeo  --EPFile  EPFile4
#  --Atom  Rb  --XiFile  ../XiFile2
# 
#
# data columns:
#1: X (um) 
#2: Y (um) 
#3: Z (um) 
#4: Xi (imaginary frequency) (3e14 rad/sec)
#5: polarizability of Rb (a_0^3)
#6: CP potential per unit frequency for Rb (neV / (3e14 rad/sec))
# scuff-caspol running on (null) (05/05/20::12:57:49)
#
# command line:
#
# scuff-caspol  --geometry  PECSlab_2640.scuffgeo  --Atom  Rb
#  --EPFile  EPFile4  --XiFile  ../XiFile2
# 
#
# data columns:
#1: X (um) 
#2: Y (um) 
#3: Z (um) 
#4: Xi (imaginary frequency) (3e14 rad/sec)
#5: polarizability of Rb (a_0^3)
#6: CP potential per unit frequency for Rb (neV / (3e14 rad/sec))
# scuff-caspol running on (null) (05/05/20::13:00:52)
#
# command line:
#
# scuff-caspol  --geometry  PECSlab_2640.scuffgeo  --Atom  Rb
#  --EPFile  EPFile4  --XiFile  ../XiFile2
# 
#
# data columns:
#1: X (um) 
#2: Y (um) 
#3: Z (um) 
#4: Xi (imaginary frequency) (3e14 rad/sec)
#5: polarizability of Rb (a_0^3)
#6: CP potential per unit frequency for Rb (neV / (3e14 rad/sec))
# scuff-caspol running on (null) (05/06/20::10:51:00)
#
# command line:
#
# scuff-caspol  --geometry  PECSlab_2640.scuffgeo  --Atom  Rb
#  --EPFile  EPFile4  --XiFile  ../XiFile2
# 
#
# data columns:
#1: X (um) 
#2: Y (um) 
#3: Z (um) 
#4: Xi (imaginary frequency) (3e14 rad/sec)
#5: polarizability of Rb (a_0^3)
#6: CP potential per unit frequency for Rb (neV / (3e14 rad/sec))
