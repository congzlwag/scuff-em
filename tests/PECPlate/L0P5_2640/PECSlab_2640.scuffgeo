OBJECT Slab
    MESHFILE Slab_L0P5_2640.msh
    MATERIAL PEC
ENDOBJECT
