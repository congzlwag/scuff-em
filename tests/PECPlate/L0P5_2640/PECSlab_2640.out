# scuff-caspol running on (null) (05/08/20::20:21:00)
#
# command line:
#
# scuff-caspol  --geometry  PECSlab_2640.scuffgeo  --Atom  Rb
#  --EPFile  EPFile4 
#
# data columns:
#1: X (um) 
#2: Y (um) 
#3: Z (um) 
#4: Casimir-polder potential for Rb (neV)
0.000000e+00 0.000000e+00 3.000000e-02 -3.116931e+02 
0.000000e+00 0.000000e+00 3.000000e-02 -3.116931e+02 
0.000000e+00 0.000000e+00 3.500000e-02 -1.920481e+02 
0.000000e+00 0.000000e+00 4.000000e-02 -1.253709e+02 
0.000000e+00 0.000000e+00 4.500000e-02 -8.568358e+01 
0.000000e+00 0.000000e+00 5.000000e-02 -6.076478e+01 
0.000000e+00 0.000000e+00 5.500000e-02 -4.441804e+01 
0.000000e+00 0.000000e+00 6.000000e-02 -3.329730e+01 
0.000000e+00 0.000000e+00 6.500000e-02 -2.549689e+01 
0.000000e+00 0.000000e+00 7.000000e-02 -1.988124e+01 
0.000000e+00 0.000000e+00 7.500000e-02 -1.574698e+01 
0.000000e+00 0.000000e+00 8.000000e-02 -1.264361e+01 
